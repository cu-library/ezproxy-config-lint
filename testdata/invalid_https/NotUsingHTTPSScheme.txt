Group General+Affiliate+Retiree
Title EBSCO Electronic Journals Service
URL http://ejournals.ebsco.com
HJ ejscontent.ebsco.com
DJ ebsco.com
HJ content.ebsco.com
