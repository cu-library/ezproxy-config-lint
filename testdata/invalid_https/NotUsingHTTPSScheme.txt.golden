testdata/invalid_https/NotUsingHTTPSScheme.txt:3: URL http://ejournals.ebsco.com ‚Üê URL is not using HTTPS scheme (L3007)
